test/cypress/features/transaction/transaction.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+import { Given, Then } from 'cypress-cucumber-preprocessor/steps';
+import { ss } from '../../../constants';
+
+Given(/^Given I am on Transactions page$/, function () {
+	cy.visit('/transactions');
+})
```

Comment:
there already exists a step implementation that takes care of page/screen visitation in `common.js` it would be better to either extend or use that step definition

![image](https://user-images.githubusercontent.com/32064769/150993161-91534c67-1cd3-4e14-8b87-1bd4fb2096ed.png)

Reply:
Fixed